lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -19,6 +19,18 @@ def initialize(vmpooler_hosts, options)
       @options = options
       @logger = options[:logger]
       @hosts = vmpooler_hosts
+      @credentials = load_credentials(@options[:dot_fog])
```

Comment:
The way this is written it will always attempt to load the .fog since beaker provides a default value for options[:dot_fog] - so, I believe, that this will now throw an error if one isn't specified.

I know that it is hacky, but we may need an additional switch param here to indicate that we are going to be running authorized so that we can avoid always attempting to open the dot fog.